packages/common/src/core/types/IHubEntityBase.ts

```diff
@@ -14,28 +14,32 @@ export interface IHubEntityBase {
    */
   name: string;
   /**
-   * Simple summary of the Entity.
-   * For items, this is typically the `snippet` property.
+   * Sanitized summary derived from item.snippet, item.description,
+   * group.description, user.description, event.description etc
    */
   summary?: string;
   /**
    * Date the entity was created
    */
   createdDate: Date;
   /**
-   * Source of the creation date
+   * Source of the creation date as a property path
+   * e.q `item.created`
    */
   createdDateSource: string;
   /**
-   * Date the entity was last updated
+   * Date when the entity was last updated
+   * Depending on the entity, this could be derived
+   * in many different ways
    */
   updatedDate: Date;
   /**
    * Source of the updated date
    */
   updatedDateSource: string;
   /**
-   * Type of entity
+   * For Item backed results, this will be `item.type`
```

Comment:
should we also have HubFamily [dataset, map, app, ...] instead of (or in addition to) the item type? 
or do we believe that this should be the original and we layer hubType elsewhere for filtering, faceting, and display?

Reply:
I think we want it to be "in addition" because it's very useful to have the backing type. 

That said, there is a note in the code that says `HubType` is deprecated and we should use `HubFamily` instead... which I see is already defined... commit inbound which removes the newly created `HubFamily` type